synapse/rest/__init__.py

```diff
@@ -117,4 +117,6 @@ def register_servlets(client_resource, hs):
         account_validity.register_servlets(hs, client_resource)
 
         # moving to /_synapse/admin
-        synapse.rest.admin.register_servlets(hs, client_resource)
+        synapse.rest.admin.register_servlets_for_client_rest_resource(
+            hs, client_resource
+        )
```

Comment:
What's this for?

Reply:
well, it comes from #5120 ...

Reply:
Ah, I didn't notice you renamed the function below, sorry.